code/modules/projectiles/projectile/special/plasma.dm

```diff
@@ -6,11 +6,21 @@
 	range = 4
 	dismemberment = 20
 	impact_effect_type = /obj/effect/temp_visual/impact_effect/purple_laser
+    var/pressure_decrease_active = FALSE
+    var/pressure_decrease = 0.2
 	var/mine_range = 3 //mines this many additional tiles of rock
 	tracer_type = /obj/effect/projectile/tracer/plasma_cutter
 	muzzle_type = /obj/effect/projectile/muzzle/plasma_cutter
 	impact_type = /obj/effect/projectile/impact/plasma_cutter
 
+/obj/item/projectile/plasma/Initialize()

```

Comment:
` /obj/item/projectile/plasma/adv/mech/Initialize()` also move it under the child instead of here.

Reply:
thank you dearly.